karma.conf.ts

```diff
@@ -5,7 +5,6 @@ export default function(config: any) {
   const chromeFlags = [
     '--headless',
     '--no-sandbox',
-    '--disable-gpu',
```

Comment:
I am removing this flag since it was causing the following error on Apple ARM processors:
```sh
ERROR LOG: 'processFrame error:', TypeError: Cannot read properties of null (reading 'canvas')
```